.github/workflows/comment_pr.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   comment-report:
+    needs: code-metrics
```

Comment:
[Question]
- Can this refer to other workflows?
- What happens if there is no `needs` tag?